zhaquirks/tuya/ts011f_plug.py

```diff
@@ -1022,3 +1032,51 @@ class Plug_4AC_2USB_Metering(EnchantedDevice):
             },
         },
     }
+
+
+class Plug_v2(CustomDevice):
```

Comment:
Might not be necessary, but ``Plug`` extends ``EnchantedDevice`` (which later extends ``CustomDevice``). 
``EnchantedDevice`` adds the "Tuya spell" which is needed for some device, but I've never needed it for plugs IIRC.

I'd still follow whatever the main ``Plug`` class does and either make ``Plug_v2`` extend ``EnchantedDevice`` or just ``Plug`` (from above) (which then extends ``EnchantedDevice``).

(@javicalle might know better what to do with the ``EnchantedDevice`` stuff)

Reply:
We shall / haw starting  moving all new devices to` EnchantedDevice` device class then it coming new devices that need it and have the same signature as current ones. I was updating all classes in one of the switch quirks and its looks working well for 2 weeks ago and its in HA 2023.1. And its needing little more housekeeping and testing for getting all device migrated but i think its needed in the long run.
One famous one is LIDL power strip new version with updated firmware is needing it very badly

Reply:
I've moved to EnchantedDevice now